src/main/java/htsjdk/samtools/cram/compression/rans/ArithmeticDecoder.java

```diff
@@ -24,25 +24,28 @@
  */
 package htsjdk.samtools.cram.compression.rans;
 
-final class ArithmeticDecoder {
-    final FC[] fc = new FC[256];
+final public class ArithmeticDecoder {
+    public final int[] freq = new int[Constants.NUMBER_OF_SYMBOLS];
+    public final int[] cumulativeFreq = new int[Constants.NUMBER_OF_SYMBOLS];
```

Comment:
I totally forgot that we made this optimization for `FC`. Should cut down on memory allocation and garbage collection.